tpch/queries/q14.py

```diff
@@ -26,7 +21,7 @@ def query(line_item_ds: FrameT, part_ds: FrameT) -> FrameT:
                 .sum()
                 / (nw.col("l_extendedprice") * (1 - nw.col("l_discount"))).sum()
             )
-            .round(2)
+            # .round(2)
```

Comment:
duckdb results is not rounded! I panicked for a second and had to check we implement it correctly